src/CoreFx.Private.TestUtilities/ref/CoreFx.Private.TestUtilities.cs

```diff
@@ -149,6 +149,7 @@ public sealed partial class RemoteInvokeOptions
         public string ExceptionFile { [System.Runtime.CompilerServices.CompilerGeneratedAttribute]get { throw null; } }
         public int ExpectedExitCode { [System.Runtime.CompilerServices.CompilerGeneratedAttribute]get { throw null; } [System.Runtime.CompilerServices.CompilerGeneratedAttribute]set { } }
         public bool Start { [System.Runtime.CompilerServices.CompilerGeneratedAttribute]get { throw null; } [System.Runtime.CompilerServices.CompilerGeneratedAttribute]set { } }
+        public bool RunAsSudo { [System.Runtime.CompilerServices.CompilerGeneratedAttribute]get { throw null; } [System.Runtime.CompilerServices.CompilerGeneratedAttribute]set { } }
```

Comment:
RunAsSudo is a very Unix specific term. As you are adding it to the refs for all platforms consider rephrasing the method.

Reply:
I choose that name as it matched the existing method in this library:  https://github.com/dotnet/corefx/blob/master/src/CoreFx.Private.TestUtilities/ref/CoreFx.Private.TestUtilities.cs#L16.  That method is also on all platforms.